code/modules/hydroponics/plants_crop.dm

```diff
@@ -1,5 +1,6 @@
 /datum/plant/bamboo
 	name = "Bamboo"
+	plant_icon = 'icons/obj/hydroponics/plants_crop.dmi'
```

Comment:
There's a lot of setting `plant_icon` in here (and sibling files). Not requiring for your PR, but as a possible future improvement for someone to pick up is breaking up the plant datums into types (similar to how the icons are broken up), e.g. `/datum/plant/crop/bamboo`. That way you could set the `plant_icon` on `/datum/plant/crop` (and the `category` var for most of them, as well) and be done with it without having to set on every single datum specifically.

Reply:
That's a solid shout - i might get that sorted here actually so it's not forgotten

Reply:
Implemented in latest commit :)